puppet/lib/puppet/reports/puppetdb.rb

```diff
@@ -73,6 +75,41 @@ def build_events_list
     end
   end
 
+  # @return Array[Hash]
+  # @api private
+  def build_logs_list
+    profile("Build logs list (count: #{logs.count})",
+            [:puppetdb, :logs_list, :build]) do
+      log_list = []
+      logs.each do |log|
+        log_hash = {
+          'file' => log.file,
+          'line' => log.line,
+          'level' => log.level,
+          'message' => log.message,
+          'source' => log.source,
+          'tags' => [*log.tags],
```

Comment:
Lol: http://stackoverflow.com/questions/9653889/coercing-a-scalar-or-an-array-to-become-an-array